config/config.go

```diff
@@ -24,23 +24,30 @@ var (
 )
 
 const Version = "v0.1.5"
+const UsageOracleKey = "Set the oracle server key file path."
+const UsagePluginConf = "Set the plugins' configuration file path."
```

Comment:
type plugins' ==> plugin's

Reply:
> type plugins' ==> plugin's

Resolved in bd07764ac76b51a6cd8ae44a169d0b74eb116d63